create3_lidar/README.md

```diff
@@ -62,36 +69,49 @@ There may be some warnings and errors on startup, but the following message will
 [async_slam_toolbox_node-1] Registering sensor: [Custom Described Lidar]
 ```
 
-In a third terminal, drive the robot around:
+In a third terminal, drive the robot around. Replace `ROBOTNAMESPACE` with the namespace of your robot. If you are not using a namespace, you may delete `--ros-args -r __ns:=/ROBOTNAMESPACE` from the command:
 
 ```bash
-ros2 run teleop_twist_keyboard teleop_twist_keyboard
+ros2 run teleop_twist_keyboard teleop_twist_keyboard --ros-args -r __ns:=/ROBOTNAMESPACE
 ```
 
 ### Computer Setup
+On your computer, install rviz2.
 
-On your computer, clone and build this, and source the setup shell scripts.
+```
+sudo apt install ros-humble-rviz2
+```
 
-![Image of rviz with custom configuration](https://iroboteducation.github.io/create3_docs/examples/data/create3_lidar_rviz.png)
+Clone and build the [create3_examples repository](https://github.com/iRobotEducation/create3_examples). 
+Then source the setup shell scripts.
+
+```
+source ~/create3_examples_ws/install/local_setup.sh
+```
 
-On your computer, start rviz2.
+
+Run the sensors launch script, which includes the LIDAR driver and transform from the laser scan to the robot. Replace `ROBOTNAMESPACE` with the namespace of your robot. If you are not using a namespace, you may delete `namespace:='ROBOTNAMESPACE'` from the command:
 
 ```bash
-rviz2
+ros2 launch create3_lidar rviz_launch.py namespace:='ROBOTNAMESPACE'
 ```
+The rviz2 configuration this command imports will configure rviz2 to subscribe to the laser, the occupancy map, and display the `base_footprint` tf frame that the laser is building off of from the map frame.
```

Comment:
We should just run rviz specifying the configuration, rather than launching the whole lidar TF thing.